### PR TITLE
Add year columns to datasets in concept model

### PR DESCRIPTION
**Pull request by pletale**
Add 'year' or 'tax_year' to the list of column names for each dataset. 
Jira ticket: https://jira.ihme.washington.edu/browse/SSCI-1465

### CODE REVIEW

File: docs/source/models/concept_models/vivarium_census_synthdata/concept_model.rst [hunk 1]
```diff
@@ -2005,6 +2005,7 @@ Census
   * - First name
   * - Middle initial
   * - Last name
+  * - Year
```
Comment (zmbc) [inline docs/source/models/concept_models/vivarium_census_synthdata/concept_model.rst:2008]: As I noted in the pseudopeople PR, let's check the order of the columns here. I think we'd want this at the beginning (maybe before even the unique IDs?)

**Review by zmbc — APPROVED** [review]

File: docs/source/models/concept_models/vivarium_census_synthdata/concept_model.rst (hunk 1 above)
Comment (NathanielBlairStahn) [reply docs/source/models/concept_models/vivarium_census_synthdata/concept_model.rst:2008]: As I noted in the [pseudopeople PR](https://github.com/ihmeuw/pseudopeople/pull/241), year/tax_year is currently the _last_ column in the datasets in which it appears. Do we like that, or do we want to move it to the beginning?

Are the column orders always consistent between the concept model, pseudopeople docs, and the actual data? I'm not sure...

Comment (NathanielBlairStahn) [reply docs/source/models/concept_models/vivarium_census_synthdata/concept_model.rst:2008]: @pletale As noted in the pseudopeople PR, let's list `year` / `tax_year` as the **last** column in the relevant datasets, to be consistent with what's already implemented.

Comment (pletale) [reply docs/source/models/concept_models/vivarium_census_synthdata/concept_model.rst]: @zmbc @NathanielBlairStahn Just moved the year columns to the last column in the datasets (Census, WIC, taxes)!

**Comment by zmbc** [comment]
@pletale is there no year column in the survey observer?

**Review by zmbc — APPROVED** [review]

**Review by NathanielBlairStahn — APPROVED** [review]

**Pull request merged**